src/o365/graph/commands/planner/planner-task-list.ts

```diff
@@ -0,0 +1,102 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+import { Task } from './Task';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+
+}
+
+class GraphPlannerTaskListCommand extends GraphItemsListCommand<Task> {
+  public get name(): string {
+    return `${commands.PLANNER_TASK_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Lists Planner tasks of the current logged in user.';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    let endpoint: string = "/me/planner/tasks";
```

Comment:
This can be simplified and defined directly inline in the `getAllItems` method call